src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -25,6 +25,7 @@ namespace Microsoft.PowerShell.Commands
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "http://go.microsoft.com/fwlink/?LinkId=799466")]
     [Alias("gin")]
+    [OutputType(typeof(ComputerInfo), typeof(PSCustomObject))]
```

Comment:
please use `typeof(PSObject)` as shown in `Get-Command`:
https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/GetCommandCommand.cs#L30